base/db/R/insert.format.vars.R

```diff
@@ -23,18 +23,18 @@
 #' bety <- PEcAn.DB::betyConnect()
 #' 
 #' formats_variables_tibble <- tibble::tibble(
-#'        variable_id = c(411, 135, 382), # integer 
-#'        name = c("NPP", NA, "YEAR"), # 
+#'        variable_id = c(411, 135, 382), 
+#'        name = c("NPP", NA, "YEAR"),  
 #'        unit = c("g C m-2 yr-1", NA, NA),
 #'        storage_type = c(NA, NA, "%Y"),
 #'        column_number = c(2, NA, 4),
 #'  )
-#'   insert.format.vars(con = bety$con, name = "LTER-HFR-103", mimetype_id = 1090, notes = "NPP from Harvard Forest.", header = FALSE, skip = 0, formats_variables = formats_variables_tibble)
+#'   insert.format.vars(con = bety$con, format_name = "LTER-HFR-103", mimetype_id = 1090, notes = "NPP from Harvard Forest.", header = FALSE, skip = 0, formats_variables = formats_variables_tibble)
```

Comment:
Yes! This test to resolve if the format name already existed was throwing errors because of this part of line 44: ```name_test <- dplyr::tbl(con, "formats") %>% dplyr::select(id, name) %>% dplyr::filter(name %in% format_name) %>% collect()```

where ` dplyr::filter(name %in% format_name)` was throwing errors when `format_name` was `name` adding the prefix prevents confusion since the BETYdb column is `name` and the query variable was name. Does this make sense?